app/views/SearchMessagesView/index.tsx

```diff
@@ -155,9 +156,26 @@ class SearchMessagesView extends React.Component<ISearchMessagesViewProps, ISear
 		// If it's not a encrypted room, search messages on the server
 		const result = await RocketChat.searchMessages(this.rid, searchText, QUERY_SIZE, this.offset);
 		if (result.success) {
-			return result.messages;
+			const urlRenderMessages = result.messages?.map(message => {
+				// @ts-ignore
+				if (message.urls.length > 0) {
+					message.urls = message.urls?.map((url, index) => {
+						if (url.meta !== undefined) {
```

Comment:
```suggestion
						if (url.meta) {
```

Reply:
@gerzonc I have made the required changes, please review, thanks